src/utils.js

```diff
@@ -28,6 +28,14 @@ function isErrorEvent(value) {
   return supportsErrorEvent() && {}.toString.call(value) === '[object ErrorEvent]';
 }
 
+function isDOMError(value) {
+  return supportsDOMError() && {}.toString.call(value) === '[object DOMError]';
```

Comment:
why is this guard needed? why can't is just be return `{}.toString.call(value) === '[object DOMError]'` ?